src/Monolog/Formatter/GrabyFormatter.php

```diff
@@ -61,18 +67,18 @@ protected function convertToString($data): string
     /**
      * Creates an HTML table row with background cellon title cell.
      *
-     * @param int    $level    Error level
+     * @param Level  $level    Error level
      * @param string $th       Row header content
      * @param string $td       Row standard cell content
      * @param bool   $escapeTd false if td content must not be html escaped
      */
-    private function addRowWithLevel(int $level, string $th, string $td = ' ', bool $escapeTd = true): string
+    private function addRowWithLevel(Level $level, string $th, string $td = ' ', bool $escapeTd = true): string
```

Comment:
This will not work with Monolog 2, will it?

Reply:
Hmm, I'm getting this error, I think I have monolog 2 installed.

    Graby\Monolog\Formatter\GrabyFormatter::addRowWithLevel(): Argument #2 ($th) must be of type string, int gi